src/superio/nuvoton/nct6687d/superio.c

```diff
@@ -21,6 +23,34 @@
 #define NCT6687D_ACPI_POWER_PREV_STATE		(2 << NCT6687D_ACPI_POWER_LOSS_CONTROL_SHIFT)
 #define NCT6687D_ACPI_POWER_USER_DEFINED	(3 << NCT6687D_ACPI_POWER_LOSS_CONTROL_SHIFT)
 
+static uint8_t get_power_on_after_fail(void)
```

Comment:
pmc_set_power_failure_state may also need this

Reply:
Hm, any suggestion where to put the implementation?  These EFI variables seem to be all over the place, so maybe the rest/new ones can be put there too eventually.  Maybe to something like `src/lib/dasharo.c`?

Reply:
I was thinking about putting the implementation in one place too. I think more appropriate place would be the `src/vendorcode/dasharo/options.c` or something like that.